src/SFA.DAS.ApprenticeCommitments.Web/Pages/Apprenticeships/HowYourApprenticeshipWillBeDelivered.cshtml

```diff
@@ -0,0 +1,103 @@
+﻿@page "/apprenticeships/{apprenticeshipId}/howyourapprenticeshipwillbedelivered"
+@model SFA.DAS.ApprenticeCommitments.Web.Pages.Apprenticeships.HowYourApprenticeshipWillBeDeliveredModel
+@{
+    ViewData["Title"] = "How your apprenticeship will be delivered";
+}
+
+<div class="govuk-width-container">
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            <partial name="ValidationSummary" />
+
+            <h1 asp-hide="@Model.ConfirmedHowApprenticeshipDelivered" class="govuk-heading-xl">@ViewData["Title"]</h1>
```

Comment:
A bug I noticed in the other pages is that we should be checking whether or not the confirmation *has* a value, not what the value is.

It will be the same for the other `asp-hide` and `asp-show` directives on the page.

```suggestion
            <h1 asp-hide="@Model.ConfirmedHowApprenticeshipDelivered.HasValue" class="govuk-heading-xl">@ViewData["Title"]</h1>
```